src/main/java/frc/robot/subsystems/swerve/PhoenixOdometryThread.java

```diff
@@ -127,17 +127,19 @@ public void run() {
       var writeLock = journalLock.writeLock();
       // NOTE (kevinclark): The toArray here in a tight loop is kind of ugly
       // but keeping up a symmetric array is too and it's probably negligible on latency.
-      var status =
```

Comment:
You might consider still reporting if status is coming back false. Seems useful for diagnostics.

Reply:
Maybe per signal down below?